tools/unmicst/macros.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0"?>
+<macros>
+    <xml name="requirements">
+        <requirements>
+            <container type="docker">labsyspharm/unmicst:@TOOL_VERSION@</container>
+            <!--
+            <requirement type="package" version="3.7">python</requirement>
+            <requirement type="package" version="1.15.0">tensorflow</requirement>
+            <requirement type="package" version="1.15.1">tensorflow-estimator</requirement>
+            <requirement type="package">cudnn</requirement>
+            <requirement type="package" version="10.0">cudatoolkit</requirement>
+            <requirement type="package" version="0.17.2">scikit-image</requirement>
+            <requirement type="package" version="1.4.1">scipy</requirement>
+            <requirement type="package" version="2020.7.24">tifffile</requirement>
+            <requirement type="package" version="2019.7.2">czifile</requirement>
+            <requirement type="package" version="3.2.3">nd2reader</requirement>
+            -->
+        </requirements>
+    </xml>
+
+    <xml name="version_cmd">
+        <version_command>echo @TOOL_VERSION@</version_command> 
+    </xml>
+    <xml name="citations">
+        <citations>
+            <citation type="doi">10.1101/2021.04.02.438285</citation>
+        </citations>
+    </xml>
+
+    <token name="@TOOL_VERSION@">2.7.1</token>
+    <token name="@VERSION_SUFFIX@">0</token>
+    <token name="@CMD_BEGIN@"><![CDATA[
+    UNMICST_PATH='' &&
+    if [ -f '/app/unmicstWrapper.py' ]; then
+        export UNMICST_PATH='python /app/unmicstWrapper.py';
```

Comment:
I have seen this pattern now a few times, question why is this script not on PATH in the container?  This would make the if else redundant isn't it?

Reply:
The goal is to set up tools that are both docker and conda compatible. However, the paths/installation process is different between the two. This chunk of code is intended to find the script for either environment.

Reply:
I see so you are not controlling the container and because of that you can not add the binary to the PATH?

Reply:
Yes exactly.